CONTRIBUTING.md

```diff
@@ -5,134 +5,51 @@
-
-
-### Suggesting Enhancements
-
-This section guides you through submitting an enhancement suggestion for CloudQuery, including completely new features, minor improvements to existing functionality and new plugins. Following these guidelines helps maintainers and the community understand your suggestion :pencil: and find related suggestions :mag_right:.
-
-Before creating enhancement suggestions, please check [this list](#before-submitting-an-enhancement-suggestion) as you might find out that you don't need to create one. When you are creating an enhancement suggestion, please [include as many details as possible](#how-do-i-submit-a-good-enhancement-suggestion). Fill in [the template](.github/ISSUE_TEMPLATE/feature_request.md), including the steps that you imagine you would take if the feature you're requesting existed.
-
-#### Before Submitting an Enhancement Suggestion
-
-* **Determine [which repository the enhancement should be suggested in](#break-down-of-responsibilities-and-repositories)**
-* **Perform a [cursory search](https://github.com/search?q=is%3Aopen+label%3Aenhancement+org%3Acloudquery)** to see if the enhancement has already been suggested. If it has, add a comment to the existing issue instead of opening a new one
-
-#### How Do I Submit a (Good) Enhancement Suggestion?
-
-Enhancement suggestions are tracked as [GitHub issues](https://guides.github.com/features/issues/). After you've determined [which repository](#break-down-of-responsibilities-and-repositories) your enhancement suggestion is related to, create an issue on that repository and provide the following information:
-
-* **Use a clear and descriptive title** for the issue to identify the suggestion
-* **Describe the problem** In detail please try and convey the workflow or functionality you are trying to implement. This will help the community design and implement tooling that is both intuitive to use across many different domains as well as applicable to you and your specific challenge
-* **Describe the use case** for this feature in as much detail as possible. Be sure to include any relevant information inluding links or other implementations
+Follow our [bug reporting template](https://github.com/cloudquery/plugin-sdk/issues/new?assignees=&labels=bug%2Cneeds-triage&template=bug_report.yml&title=%28short+issue+description%29) or [feature request template](https://github.com/cloudquery/plugin-sdk/issues/new?assignees=&labels=feature-request%2Cneeds-triage&template=feature_request.yml&title=%28short+issue+description%29) to ensure you provide all the necessary information for us to either reproduce and fix the bug or implement the feature.
 
 ### Your First Code Contribution
 
-Unsure where to begin contributing to CloudQuery? You can start by looking through these `beginner` and `help-wanted` issues:
```

Comment:
We haven't used `beginner` anywhere. Also I think searching for 1 label should be enough. `good first issue` is a default label all GitHub repositories have